### PR TITLE
[#5114] Improvement:use incubating version in docs

### PR DESCRIPTION
**Pull request by lsyulong**
### What changes were proposed in this pull request?

[#5114] Improvement:use incubating version in docs

### Why are the changes needed?

Fix: #5114 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

No need

### CODE REVIEW

File: docs/how-to-use-gvfs.md [hunk 1]
```diff
@@ -335,18 +335,17 @@ to recompile the native libraries like `libhdfs` and others, and completely repl
...
-| `oauth2_scope`       | The auth scope for the Gravitino client when using `oauth2` auth type with the Gravitino Virtual File System.                                            | (none)        | Yes if you use `oauth2` auth type | 0.7.0         |.
-
+| Configuration item   | Description                                                                                                                                              | Default value | Required                          | Since version    |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|-----------------------------------|------------------|
+| `server_uri`         | The Gravitino server uri, e.g. `http://localhost:8090`.                                                                                                  | (none)        | Yes                               | 0.6.0-incubating |.                                                                                                                | (none)        | Yes                                 | 0.6.0         |
```
Comment (yuqi1129) [inline docs/how-to-use-gvfs.md:340]: Please remove this `0.6.0` here.

**Comment by jerryshao** [comment]
Thanks @lsyulong for your work.

File: docs/how-to-use-gvfs.md (hunk 1 above)
Comment (lsyulong) [reply docs/how-to-use-gvfs.md:340]: image

Do you mean to write it like this on the line 'server uri'？

Comment (yuqi1129) [reply docs/how-to-use-gvfs.md:340]: I mean why there still version `0.6.0` in the document

Comment (lsyulong) [reply docs/how-to-use-gvfs.md:340]: ok, i'll check my code

Comment (jerryshao) [reply docs/how-to-use-gvfs.md:340]: There are some issues in the current doc, we just fixed the issue, @lsyulong can you please rebase your code, the comment here will be addressed automatically after you rebase the code against the main branch.

**Comment by jerryshao** [comment]
Just rebase your branch against main branch, then solve the conflicts, that should be enough.

**Comment by lsyulong** [comment]
> Just rebase your branch against main branch, then solve the conflicts, that should be enough.

Sorry, due to network issues, I submitted multiple revisions during the process, which caused inconvenience to the teacher

**Comment by jerryshao** [comment]
Seems like you still need to fix the conflicts before it can be merged.

**Comment by jerryshao** [comment]
@yuqi1129 can you please help to solve this conflict here?

**Comment by yuqi1129** [comment]
> till need to fix the conflicts before it

OK

File: docs/how-to-use-gvfs.md (hunk 1 above)
Comment (yuqi1129) [reply docs/how-to-use-gvfs.md]: The code has been revised.

File: docs/how-to-use-gvfs.md [hunk 2]
```diff
@@ -335,18 +335,17 @@ to recompile the native libraries like `libhdfs` and others, and completely repl
...
+| `oauth2_server_uri`  | The auth server URI for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |.
+| `oauth2_credential`  | The auth credential for the Gravitino client when using `oauth2` auth type.                                                                              | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |.
+| `oauth2_path`        | The auth server path for the Gravitino client when using `oauth2` auth type. Please remove the first slash `/` from the path, for example `oauth/token`. | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |.
+| `oauth2_scope`       | The auth scope for the Gravitino client when using `oauth2` auth type with the Gravitino Virtual File System.                                            | (none)        | Yes if you use `oauth2` auth type | 0.7.0-incubating |.
```
Comment (jerryshao) [inline docs/how-to-use-gvfs.md:348]: We still have one more column `.`, @yuqi1129 please help to fix it and carefully check it.

**Review by jerryshao — APPROVED** [review]
LGTM, thanks @laiyousin for your work, thanks @yuqi1129 for your help.

**Pull request merged**